contracts/test/MockCore.sol

```diff
@@ -40,19 +40,19 @@ contract MockCore is Core {
 	{}
 	
 	/**
-	  * @notice get storage root for a given block height
+	  * @notice Get the mocked storage root for a given block height.
 	  *
-	  * @dev this is for testing only so the data is mocked here
+	  * @dev This is for testing only so the data is mocked here.
 	  *
-	  * @param _blockHeight block height for which storage root is needed
+	  * @param _blockHeight Block height for which storage root is needed.
 	  *
-	  * @return bytes32 storage root
+	  * @return bytes32 Mocked value of storage root.
 	  */
 	function getStorageRoot(
 		uint256 _blockHeight
 	)
 		public
-		view
+		pure
```

Comment:
Did you change this to fix compiler warning? 🤔  
If yes, you probably want to fix same for `getStateRoot` also, right?

Reply:
Compiler was not warning this statement. 
Apparantly while fixing `getStateRoot` I realised that when it is changed to `pure` it causes compilation error as its overridden function. So I just made it a proper view function. It used just dummy data.

Also I removed `getStorageRoot` it was not used anywhere.